jenkins-scripts/lib/windows_env_vars.bat

```diff
@@ -11,10 +11,14 @@ if DEFINED MAKE_JOBS (
   set VCPKG_MAX_CONCURRENCY=%MAKE_JOBS%
 )
 set VCPKG_OSRF_DIR=%VCPKG_DIR%\osrf_vcpkg_ports
+set VCPKG_INSTALLED_FILES_DIR=%VCPKG_DIR%\installed
 set VCPKG_CMD=%VCPKG_DIR%\vcpkg.exe
 set VCPKG_CMAKE_TOOLCHAIN_FILE=%VCPKG_DIR%/scripts/buildsystems/vcpkg.cmake
 if NOT DEFINED VCPKG_SNAPSHOT (
   :: see https://github.com/microsoft/vcpkg/releases
   set VCPKG_SNAPSHOT=2022.02.23
 )
+:: Set of common gz dependencies expected up to Garden
+set VCPKG_DEPENDENCIES_LEGACY=assimp boost-core bullet3 ccd console-bridge cppzmq cuda curl dlfcn-win32 eigen3 fcl ffmpeg freeimage gdal gflags glib gts jsoncpp libyaml libzip ogre ogre2 ogre22 openssl protobuf pybind11 qt5 qt5-winextras qwt sqlite3 tinyxml2 zeromq

```

Comment:
If I understand correctly, the per-package dependencies have all been dropped in favor of this single list of dependencies. I think that's fine because the packaging for Windows is done using conda-forge and so there's limited utility in using the vcpkg installations to try and smoke out missing dependencies.

Reply:
> If I understand correctly, the per-package dependencies have all been dropped in favor of this single list of dependencies. I think that's fine because the packaging for Windows is done using conda-forge and so there's limited utility in using the vcpkg installations to try and smoke out missing dependencies.

This is correct.